certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk3226/elements/QuestionDiagnos.java

```diff
@@ -154,6 +154,8 @@ public static ElementSpecification questionDiagnos(
         .pdfConfiguration(
             PdfConfigurationDiagnoses.builder()
                 .prefix(PDF_DIAGNOSIS_FIELD_ID_PREFIX)
+                .maxLength(172)
```

Comment:
Wasn't the asked solution that we didn't have a max limit and instead decided if the text fit in the field dynamically?

Reply:
Forvaltningen has clarified that they are satisfied with the current solution, i.e. a max limit for text fields.